redLayerModule.py

```diff
@@ -368,6 +368,7 @@ def removeSketchesAction(self):
                     self.iface.mapCanvas().scene().removeItem( sketch[3] )
                 except:
                     pass
+        self.removeAllAnnotations()
```

Comment:
Why are you removing all annotations when using eraser tool? It should erase only the pointed annotation, linked to the erasing segment and pointed by sketch[3] see line 368